<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tb1Signal">
       <attribute name="title">
        <string>Signal</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
        <item row="1" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
          <item>
           <widget class="QLabel" name="lbl_signalSettings">
            <property name="text">
             <string>Graph settings</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="1">
             <widget class="QLineEdit" name="line_signalStep">
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lbl_signalCount">
              <property name="text">
               <string>Count (K):</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="line_signalCount"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lbl_signalAlfa">
              <property name="text">
               <string>Var. α:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="line_signalAlfa"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lbl_signalBeta">
              <property name="text">
               <string>Var. β:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="line_signalBeta"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="lbl_signalMu">
              <property name="text">
               <string>Var. μ:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="line_signalMu"/>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushBtn_signaReset">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QPushButton" name="pushBtn_signalLoad">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Load</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLabel" name="lbl_signalentropyValue">
              <property name="text">
               <string>Histogram is not load</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_signalStep">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Step (Δx):</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="lbl_signalSigma">
              <property name="text">
               <string>Var. σ:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="line_signalSigma"/>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="lbl_signalEntropy">
              <property name="text">
               <string>Entropy value:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
          <item>
           <widget class="QLabel" name="lbl_signalHist">
            <property name="text">
             <string>Distribution histogram </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCustomPlot" name="wdg_signalHist" native="true"/>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="lbl__signalHistSettings">
            <property name="text">
             <string>Histogram settings</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="labelAlignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_signalHistCount">
              <property name="text">
               <string>Count:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="line_signalHistCount"/>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushBtn_signalHistReset">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushBtn_signalHistLoad">
              <property name="text">
               <string>Load</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="lbl_signalHistError">
              <property name="palette">
               <palette>
                <active>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>170</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Text">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>170</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>170</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Text">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>170</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="96">
                    <red>164</red>
                    <green>166</green>
                    <blue>168</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Text">
                  <brush brushstyle="SolidPattern">
                   <color alpha="96">
                    <red>164</red>
                    <green>166</green>
                    <blue>168</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lbl_signalHistProbability">
            <property name="text">
             <string>Probability</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tblWdg_signalHist"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
          <item>
           <widget class="QLabel" name="lbl_signalGraph">
            <property name="text">
             <string>Signal graph</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCustomPlot" name="wdg_signalGraph" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tb2Noise">
       <attribute name="title">
        <string>Gaussian noise</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1,0">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Gaussian noise graph</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Distribution histogram </string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Entropy value</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QCustomPlot" name="widget_3" native="true"/>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QCustomPlot" name="widget_4" native="true"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="2" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tb3Convolution">
       <attribute name="title">
        <string>Convolution</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Convolution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCustomPlot" name="widget_7" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tb4SignalNoise">
       <attribute name="title">
        <string>Signal + noise</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0" columnstretch="1,1,0">
        <item row="0" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Entropy value</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QCustomPlot" name="widget_5" native="true"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Distribution histogram </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QCustomPlot" name="widget_6" native="true"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Signal + noise graph</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="2" column="2">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>349</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot_lib/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
